WalletWasabi/Bases/PeriodicRunner.cs

```diff
@@ -101,10 +101,9 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 					{
 						lock (TriggerLock)
 						{
-							if (TriggeringCts.IsCancellationRequested)
+							if (TriggeringCts is { } && TriggeringCts.IsCancellationRequested)
```

Comment:
```suggestion
							if (TriggeringCts?.IsCancellationRequested)
```

Reply:
This does not compile. This was my first idea too. However, it has occurred to me just now that I can do:  `if (TriggeringCts?.IsCancellationRequested ?? false)`